frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -29,21 +29,21 @@ const LineAreaBarChart = ({
       color: getColor("text-light"),
       ticks: {
         color: getColor("text-medium"),
-        fontSize: 11,
+        fontSize: 12,
```

Comment:
Update font size according to [the new design](https://www.notion.so/metabase/Further-polish-static-viz-and-add-support-for-new-charts-29e3a1d2c55a4bf39dc0b1e5a0b2d67f#82f746429a6042f489737862d5d6db97)